forge-game/src/main/java/forge/game/ability/effects/ChooseTypeEffect.java

```diff
@@ -58,6 +60,14 @@ public void resolve(SpellAbility sa) {
                             }
                         }
                     }
+                } else if (sa.hasParam("MostPrevalentInDefinedDeck")) {
+                    Player definedP = AbilityUtils.getDefinedPlayers(
+                            sa.getHostCard(), sa.getParam("MostPrevalentInDefinedDeck"), sa).get(0);
+                    CardCollectionView deck = definedP.getAllCards();
```

Comment:
it says library not deck?

Reply:
I think I've fixed it 😉

Reply:
params no longer match, you missed a file?

Reply:
Yep, just caught it - sorry